ledger-core/v2/application/builtin/proxy/testwallet/testwallet.go

```diff
@@ -110,7 +111,8 @@ func (r *Wallet) GetPrototype() (reference.Global, error) {
 		var ret1 *foundation.Error
 		ret[1] = &ret1
 
-		res, err := common.CurrentProxyCtx.CallMethod(r.Reference, false, false, "GetPrototype", make([]byte, 0), PrototypeReference)
+		res, err := common.CurrentProxyCtx.CallMethod(
```

Comment:
looks like you do it accidentally =)

Reply:
nope, the line width exceeded screen, so I've made new line here

Reply:
ok. it looks with additional newline. my bad